src/sqlancer/databend/ast/DatabendConstant.java

```diff
@@ -318,7 +322,8 @@ public static class DatabendDateConstant extends DatabendConstant {
         public String textRepr;
 
         public DatabendDateConstant(long val) {
-            Timestamp timestamp = new Timestamp(val);
+            long t = val % 250000000000000l;
```

Comment:
The formatting tests fail (see `mvn verify -DskipTests=true`):
> Error:  src/sqlancer/databend/ast/DatabendConstant.java:[325,28] (misc) UpperEll: Should use uppercase 'L'.
> Error:  src/sqlancer/databend/ast/DatabendConstant.java:[361,28] (misc) UpperEll: Should use uppercase 'L'.

While fixing this, could you also extract the constant to a named one that explains its purpose? I guess it's some kind of truncation factor?

After this, we can merge the change. Thank you!

Reply:
I have add some comments, thanks for your review @mrigger